build-ext.sh

```diff
@@ -1,11 +1,15 @@
+#!/bin/sh
 echo "🛠  Installing dependencies."
 yarn
 echo "🛠  Building internal packages."
 yarn lerna run build --scope @stacks/connect-ui
 echo "🛠  Compiling extension."
-yarn lerna run prod:ext
+cd packages/app && yarn lerna run prod:ext
 echo "🛠  Packaging Browser Extension"
-cd packages/app/dist
-zip -r ../../../extension.zip *
+cd dist
+TS=$(date +%Y)$(date +%m)010000
+find -print | while read file; do
+    touch -t $TS "$file"
+done
```

Comment:
@wileyj can you explain what's happening here? I understand that you're changing the timestamps to make the checksum the same - that's perfect. I just don't follow this shell code!

Reply:
Sure!
Essentially, in the `dist` dir it's getting a listing of all the files using OS `find` command. 
After the listing is created, it pipes to a loop where it reads the output line by line (where each line is a file), and it then resets the timestamp to `<currentyear><current month>010000`.

As an example, here's the a similar command from a linux container:
```
# find /lib -print | while read file; do echo "file: $file"; done
file: /lib
file: /lib/mdev
file: /lib/libz.so.1
file: /lib/sysctl.d
...
```

Reply:
There are lots of ways to do this using find, this method simply seemed like the easiest (but arguably a little opaque)

Reply:
Thanks, very helpful!